corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -46,6 +46,8 @@
 
                         alert_user("{% trans 'Password changed successfully' %}", 'success');
                         ga_track_event("Edit Mobile Worker", "Reset password", "{{couch_user_id|escapejs}}");
+                    } else {
+                        alert_user("{% trans 'Password was not changed' %}", 'danger');
```

Comment:
Not a great error. we don't currently show an error even when the passwords don't match with each other. would take a bit more refactoring that gives an exact error to the user.